app_server/test/controllers/StreamController.js

```diff
@@ -155,20 +158,22 @@ lab.experiment('StreamController Tests', function () {
                       res.result.appInstance,
                       res.result.streamId));
         Code.expect(res.result.title).to.equal(streamInfo.title);
+        Code.expect(res.result.streamer.username).to.equal(bob.username);
         done();
       });
     });
   });
 
-  lab.test('Get stream by streamId invalid non-existing stream', function (done) {
-    Router.inject({method: 'GET', url: '/api/streams/3388ffff-aa00-1111-' +
-                                       'a222-00000044888c',
-                   credentials: testAccount}, function (res) {
-      Code.expect(res.result.statusCode).to.equal(404);
-      Code.expect(res.result.message).to.equal('Stream not found');
-      done();
+  lab.test('Get stream by streamId invalid non-existing stream',
+    function (done) {
+      Router.inject({method: 'GET', url: '/api/streams/3388ffff-aa00-1111-' +
+                                         'a222-00000044888c',
```

Comment:
This invalid test case is referring to a non-existing stream. I was trying to force it to make it look up nothing on the database instead of being rejected cause of validation issues.